src/Mod/Fem/App/FemPostPipelinePyImp.cpp

```diff
@@ -288,6 +288,29 @@ PyObject* FemPostPipelinePy::holdsPostObject(PyObject* args)
     return Py_BuildValue("O", (ok ? Py_True : Py_False));
 }
 
+PyObject* FemPostPipelinePy::renameArrays(PyObject* args)
+{
+    PyObject* pyObj;
+    if (!PyArg_ParseTuple(args, "O!", &(PyDict_Type), &pyObj)) {
+        return nullptr;
+    }
+
+    Py::Dict pyNames {pyObj};
+    std::map<std::string, std::string> names {};
+    for (auto it = pyNames.begin(); it != pyNames.end(); ++it) {
+        const auto& value = *it;
+        if (!value.first.isString() || !static_cast<Py::Object>(value.second).isString()) {
+            PyErr_SetString(PyExc_TypeError, "Names must be string objects");
+            return nullptr;
+        }
+        names[value.first.as_string()] = static_cast<Py::Object>(value.second).as_string();
+    }
```

Comment:
I think this can be refactored as:
```cpp
    for (auto&& [key, value] : pyNames) {
        if (!key.isString() || !value.isString()) {
            PyErr_SetString(PyExc_TypeError, "Names must be string objects");
            return nullptr;
        }
        names.emplace(key.as_string(), value.as_string());
    }
```

What do you think?